public/xmlns/manifest.xsd

```diff
@@ -16,6 +16,40 @@
     <xs:import namespace="https://mobile-content-api.cru.org/xmlns/shareable" schemaLocation="shareable.xsd" />
     <xs:import namespace="https://mobile-content-api.cru.org/xmlns/tract" schemaLocation="tract.xsd" />
 
+    <xs:annotation>
+        <xs:documentation>
+            Content Stacks
+            --------------
+
+            All of the content within the content spec is built around the concept of a content stack. A content stack
+            is a vertical stack of content that supports vertical scrolling. We try to avoid doing absolute positioning
+            of content to avoid responsive issues with differing screen sizes.
+
+            Throughout the content spec we will refer to an active content stack to reference the current content stack
+            being rendered on the screen. For example for a tract tool showing a card, the active content stack refers
+            to all content elements that are in that card.
+
+            Events and Listeners
+            --------------------
+
+            Throughout the content spec there are elements that trigger events and listeners that listen for events.
+            Whenever an element triggers an event or a series of events, each event triggered is processed in sequence.
+            For each event any "active" listeners that match the element will fire. Once each triggered event is
+            processed, the next triggered event will be processed with the updated "active" listeners. Unless otherwise
+            specified content visibility attributes have no impact on whether a listener is "active".
```

Comment:
I added a statement here that listeners aren't affected by content visibility attributes unless otherwise specified